packages/react-core/src/components/Badge/Badge.tsx

```diff
@@ -15,12 +17,14 @@ export const Badge: React.FunctionComponent<BadgeProps> = ({
   isRead = false,
   className = '',
   children = '',
+  screenReaderText,
   ...props
 }: BadgeProps) => (
   <span
     {...props}
     className={css(styles.badge, (isRead ? styles.modifiers.read : styles.modifiers.unread) as any, className)}
   >
+    {screenReaderText && <span className="pf-screen-reader">{screenReaderText}</span>}
```

Comment:
I think I agree with @thatblindgeye, let's move the screenreader text to after the badge's children.
And can we update the examples - at least the unread examples - to demonstrate recommended practices?